Store vehicles to rebel outposts

wersal454 opened the pull request:
## What type of PR is this?
1. [ ] Bug
2. [x] Change
3. [x] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

Also garage vehicles

You can now choose if vehicle should be from garage or from the store.
Oh and also you can now choose should vehicle be parallel to the road or facing the barricade.

Outpost now saves the vehicle, it's customisation and direction.

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

wersal454 committed:
Update fn_outpost_createAa.sqf

wersal454 committed:
working version

only roadblock

wersal454 committed:
working version

only roadblock

wersal454 committed:
almost done

wersal454 committed:
almost done

wersal454 commented:
Almost done

Things to figure out:
1) Price of the outposts
2) maybe add an option to choose different sandbags/structures surrounding MG, AT and AA outposts, so for example low mg's won't shoot into sandbags
3) probably a really bad idea, but maybe mortar outposts? 
4) probably forgotten something

wersal454 commented:
also would need to update this PR when additional garage categories come out

wersal454 commented:
Also, need to figure out how to put AI in the commander seat

probably similar way as in #376

wersal454 committed:
cleanup

wersal454 committed:
cleanup

wersal454 commented:
https://drive.google.com/file/d/1nrXzcxePCZN3FsLic6W6R_BYbBUz4Myf/view?usp=drive_link video that explains (hopefully) what this PR does

wersal454 commented:
> Almost done
> 
> Things to figure out:
> 
> 1. Price of the outposts
> 2. maybe add an option to choose different sandbags/structures surrounding MG, AT and AA outposts, so for example low mg's won't shoot into sandbags
> 3. probably a really bad idea, but maybe mortar outposts?
> 4. probably forgotten something

2. instead of choosing what type of building to put, could probably spawn small or medium building box so players can create outpost the way they want
Or by using building system, move camera to future outpost position, preset buildings and when HC group arrives, spawn them

wersal454 committed:
navigation help

wersal454 committed:
navigation help

wersal454 committed:
populate commander seat and every turret

wersal454 committed:
populate commander seat and every turret

wersal454 committed:
fixes

+backcompatability with older saves
+remove selected vehicle from garage

wersal454 committed:
fixes

+backcompatability with older saves
+remove selected vehicle from garage

wersal454 committed:
cleanup

wersal454 committed:
cleanup

jwoodruff40 committed:
xxxFromGarage consolidation and minor cleanup

jwoodruff40 committed:
xxxFromGarage consolidation and minor cleanup

jwoodruff40 committed:
Start consolidating xxxFromStore

jwoodruff40 committed:
Start consolidating xxxFromStore

Aranov commented:
Tested this as I liked the idea, but have two notes. 
First your current version is creating AA and AT posts as HMG posts when buying from the store. Both their respective functions references the createhmg function at end of line 113.

Second, It appears that the statics are not saving at their posts, being replaced by generic titan launchers, at least for me.

jwoodruff40 committed:
xxxFromStore mostly consolidated

jwoodruff40 committed:
xxxFromStore mostly consolidated

jwoodruff40 committed:
xxxFromStore doned

jwoodruff40 committed:
xxxFromStore doned

jwoodruff40 committed:
minor cleanup

jwoodruff40 committed:
minor cleanup

jwoodruff40 committed:
more cleanup

jwoodruff40 committed:
more cleanup

jwoodruff40 committed:
Parallel - yes / no

jwoodruff40 committed:
Parallel - yes / no

wersal454 committed:
Update fn_NATOinit.sqf

wersal454 committed:
better dud nvg detection

wersal454 committed:
fixes + garage outpost vehicle during disband

wersal454 committed:
fixes + garage outpost vehicle during disband

wersal454 committed:
I forgor

same disband logic for the rest of the outpost types

wersal454 committed:
I forgor

same disband logic for the rest of the outpost types

wersal454 committed:
wrong PR

Silence-112 commented:
Issues I noticed in local testing:

- There is still random russian in the sidechat

- The garage preview is very confusing, given it only shows the garage background when you click on a vehicle
![image](https://github.com/user-attachments/assets/53d8e039-10fd-4a99-b21f-39bcb583e3d7)

- The "Buy vehicle" UI is off-centered (1920x1080, 16:9, small interface)
![image](https://github.com/user-attachments/assets/eb486076-40e7-419b-b8d2-8d0142d518e7)

- If you don't select a vehicle in the garage UI, it will still allow you to continue and error out
> ```sqf
> 14:37:15 "Совпадений не обнаружено"
>14:37:15 Error in expression <eFrom, _x];
>};
>} forEach HR_GRG_Vehicles#_CategoryToremoveVehicleFrom;
>
>
>if (cou>
>14:37:15   Error position: <#_CategoryToremoveVehicleFrom;
>
>
>if (cou>
>14:37:15   Error Zero divisor
>14:37:15 File x\A3A\addons\scrt\Outpost\fn_outpost_createRoadblock.sqf..., line 125
>```

- The detection for selected garage vehicle is questionable, the vehicle may be highlighted when infact it is not selected (leading to the previous issue)

- After loading my save, the roadblocks now use the default HMG offroad and their direction hasn't saved (One of them ate my tank `D:`)

wersal454 committed:
no garage vehicles backup and no select fix

wersal454 committed:
no garage vehicles backup and no select fix

wersal454 committed:
no garage vehicles backup and no select fix

wersal454 committed:
no garage vehicles backup and no select fix

wersal454 commented:
added message and forced store selection if garage categories are empty
added automatic selection of the first entry in garage category list when you select a category
now if you don't select a vehicle (will only happen when open you garage dialog and don't click on category (this happens with regular garage aswell, don't know a good fix for that)) select button won't do anything, so you won't get an error

didn't fix : 
~~outpost vehicles not being saved, and get replaced by default vehicles~~
~~+something to do with spawner not saving vehicle info, even though said info beig transferred to spawner~~
+ui scaling issues, tried a lot of things, even tried copying dialog config from HQ store, and for whatever reason, it made it even worse
maybe @jwoodruff40 will have some ideas about UI

UnseenKill committed:
https://github.com/BrettMayson/HEMTT/issues/1041

UnseenKill committed:
Make rebel and enemy capture code shared

UnseenKill committed:
Added parameter for anti flag-yoinking experimental feature

UnseenKill committed:
Implement extended radius and zone check

UnseenKill committed:
Implement capture/defense additional diameter

UnseenKill committed:
omg

UnseenKill committed:
Added debug info

UnseenKill committed:
Added ACE inspired common interactions mechanism

UnseenKill committed:
`_vehicle` can never be `nil`

UnseenKill committed:
Code rewrite for interaction mechanism

UnseenKill committed:
Added single- & multi-use lockpick kits

UnseenKill committed:
React correctly if no lockpicks available

UnseenKill committed:
𝔖𝔭𝔯𝔦𝔠𝔥𝔰𝔱 𝔇𝔲 𝔇𝔢𝔲𝔱𝔰𝔠𝔥

UnseenKill committed:
Implemented random lockpick breakage

UnseenKill committed:
We're not that far, yet

UnseenKill committed:
Better nomenclature

UnseenKill committed:
Know your prefixes

UnseenKill committed:
code review

UnseenKill committed:
ixnay on the rpt amspay

UnseenKill committed:
Make review easier

UnseenKill committed:
Reduce zone check server load by delayed checks for marker & size

UnseenKill committed:
Ensure function returns even when a callback croaks

UnseenKill committed:
Differentiate between wait and actual execution

UnseenKill committed:
Comments

wersal454 committed:
dummy nvg array

wersal454 committed:
cleanup + lighter real nvg selection

MoerasGrizzly committed:
3CBF Vanilla Civilian Template

This adds a template for vanilla civilians with 3cbf vehicles.

Works quite nicely on all the vanilla maps, and should be useful enough since there's a distinct lack of templates that work well for Altis, Malden and Tanoa.

Grusey1 committed:
Added more CUP and SFP cargo nodes

Grusey1 committed:
Added more CUP and SFP cargo nodes. Fixed some existing vehicles and missing vehicles

UnseenKill committed:
Code review

UnseenKill committed:
Moved feature config from "patches"

UnseenKill committed:
merge artifact

UnseenKill committed:
punctuation

UnseenKill committed:
Add lockpicks to store

UnseenKill committed:
Fixed include path

UnseenKill committed:
Include fixes

UnseenKill committed:
Really.

UnseenKill committed:
Param order

UnseenKill committed:
Error Undefined variable in expression: markersimmune File x\A3A\addons\ultimate\functions\zones\fn_revealRandomZones.sqf, line 45

UnseenKill committed:
in-game fixes

UnseenKill committed:
Final code review

UnseenKill committed:
When in vehicle, don't show interactions by default

UnseenKill committed:
Author correction

Silence-112 committed:
Add: Disable virtual, physical mortar params

jwoodruff40 committed:
Don't punish throwing grenades at HQ if under attack

jwoodruff40 committed:
Lockpicking param for civilian vehicles

atomsky committed:
Added save and load functionality for enemy Radio Keys

atomsky committed:
Styling fixed

Silence-112 committed:
Fix: Carrying/attached logic

Majority of this is a port of:
https://github.com/official-antistasi-community/A3-Antistasi/pull/3210

jwoodruff40 committed:
Add localizations for placeholders in various stringtables

UnseenKill committed:
Repaired params

UnseenKill committed:
Fixed weird am artifacts

UnseenKill committed:
Some stringtable consistency

UnseenKill committed:
Save vehicle customization for "parked" vehicles and statics

UnseenKill committed:
Function can be undefined

UnseenKill committed:
Missed a spot

UnseenKill committed:
Don't look at game logic

UnseenKill committed:
Duuuh

jwoodruff40 committed:
Fix mission request break when no near markers for conquest

jwoodruff40 committed:
don't open commander or battle menu while in zeus or arsenal

UnseenKill committed:
Moved lockpick items to new store category "Antistasi Ultimate"

UnseenKill committed:
Make items available for Zeus, too

MoerasGrizzly committed:
Update A3A/addons/core/Templates/Templates/3CBF/templates.hpp

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: MoerasGrizzly <moerasgrizzly@protonmail.com>

Silence-112 committed:
Inverted logic fix

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: Silence <78276788+SilenceIsFatto@users.noreply.github.com>

wersal454 committed:
Update A3A/addons/core/functions/CREATE/fn_NATOinit.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update fn_NATOinit.sqf

wersal454 committed:
Update fn_NATOinit.sqf

wersal454 committed:
Update A3A/addons/core/functions/CREATE/fn_NATOinit.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/core/functions/CREATE/fn_NATOinit.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/core/functions/CREATE/fn_NATOinit.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

UnseenKill committed:
Only do the whole iteration thing if there's something to call back.

UnseenKill committed:
Distribute points for any friendly player in legacy area or new area

atomsky committed:
Prevent automatic rearm after surrender

If weapon is dropped before going unconscious, ACE routine will try to re-equip it after surrender ignoring all AI behaviors disabled. Fixed by copying weapon into surrender box and disposing of original thus preventing the action.

jwoodruff40 committed:
persist only non-default (flipped) static manning state

jwoodruff40 committed:
fix staticsToFlip logic based on param setting

jwoodruff40 committed:
re-enable AI static mortar manning

jwoodruff40 committed:
fix param localization

jwoodruff40 committed:
Version 11.8.7

jwoodruff40 committed:
Fix min items calculation in AI arsenal (only caused issues with loaded mag tabs)

jwoodruff40 committed:
fix conditional

jwoodruff40 committed:
misc fixes for grenadier loadouts

jwoodruff40 committed:
remove randomize uniforms button (for now)

atomsky committed:
German names and currency

German names for GM/CUP BW/BWA3 factions and DM as currency on Weferlingen

jwoodruff40 committed:
Version 11.8.8

atomsky committed:
DM Currency added to the GM Civ template

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: TEH Megamin <infernally.good@gmail.com>

Lentomakkara committed:
Update rhs.hpp

adds RHS and 3CBF AFRF AK-74M variants, some AK magazines, and MANPADS ammo.

Signed-off-by: Lentomakkara <tuomaspuu@protonmail.com>

UnseenKill committed:
Wrong parameter count for VARDEF

UnseenKill committed:
So that happened

Silence-112 committed:
Update: AMF tiers, fix incorrect assets

Silence-112 committed:
Add: Arid AMF faction

Silence-112 committed:
Fix: Arms Dealer Patchname

Silence-112 committed:
Fix: Missing NIArms in requiredAddons[]

Silence-112 committed:
Update: Arms Dealer (Vehicles, Weapons)

Silence-112 committed:
Fix: UK3CB classnames in RHS

Silence-112 committed:
Add: Linear/granular spawning

UnseenKill committed:
No more annoying "Destroy" popups all around base

UnseenKill committed:
Better variable name, added localization

vsbyt committed:
FWA trader additions + rebalance

vsbyt committed:
Update fwa.hpp

vsbyt committed:
Update fwa.hpp

Silence-112 committed:
Tweak: New fireteam composition

UnseenKill committed:
I wish we could use HEMTT

Silence-112 committed:
Add: Translations for stringtable entries

@wersal454

essem-451 committed:
Update RACS_AI_Army.sqf

added missing vehicles, and some units missing equipment

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_SLA.sqf

reworked militia completely, to be accurate to the one in the mod's faction, added vehicles, fixed some equipment 

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update cfgPlaneLoadouts.hpp

updated PRACS plane loadouts, CAS and CASDIVE definitions should be right at the start of both while AA ones are at the very end (ps gotta delete old ones somewhere in the old loadouts)

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update fn_setPlaneLoadout.sqf

deleted outdated PRACS loadouts

Signed-off-by: essem-451 <g1234500000p@gmail.com>

Silence-112 committed:
Fix: Incorrect AMF desert classnames

vsbyt committed:
Update config.hpp w/new FWA categories

UnseenKill committed:
User actions come before "move this asset"

Azodion committed:
Initial Commit

Azodion committed:
SF Muzzle Weights

vsbyt committed:
Update fwa.hpp

Azodion committed:
Elite Weapon Additions

Azodion committed:
static AA fix

essem-451 committed:
Update fn_setPlaneLoadout.sqf

added   back: (needs to be tested in game if it works)          
        {
            Error_1("Plane type %1 currently not supported for CASDIVE, please add the case!", typeOf _plane);


Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_Army.sqf

added back script component

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_Army.sqf

fixed mortar

Signed-off-by: essem-451 <g1234500000p@gmail.com>

Silence-112 committed:
Add: Minor vehicle refinements

essem-451 committed:
Update RACS_AI_Army.sqf

fixed duplicate traitor/unarmed

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_Army.sqf

added elite/mil squadleaders missing headgear

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_SLA.sqf

fixed various stuff from the reviews

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_Army.sqf

removed the static grenade launcher (mb there)

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update cfgPlaneLoadouts.hpp

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_Army.sqf

fixed flag and howitzer ammo issues

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_SLA.sqf

fixed flag

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update templates.hpp

fixed flags being the marker ones

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update RACS_AI_SLA.sqf

forgot to add mig-29 despite it even having pylon settings in configplanes

Signed-off-by: essem-451 <g1234500000p@gmail.com>

essem-451 committed:
Update templates.hpp

 mb forgot about queen's battalion flag

Signed-off-by: essem-451 <g1234500000p@gmail.com>

wersal454 committed:
underwater action for boxes

wersal454 committed:
Update Vanilla.hpp

Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update CfgVehicles.hpp

Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
WS update

wersal454 committed:
LDF hotfix (?)

wersal454 committed:
Update fn_initObjectRemote.sqf

Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Localisation for addActions for buyable items

jwoodruff40 committed:
don't break resourceCheck with no miladmins

UnseenKill committed:
Create an area around no-size friendly markers to allow saving of placed static emplacements

UnseenKill committed:
"Buildings" inherited from ThingX loaded, but never saved again

essem-451 committed:
Update RACS_AI_SLA.sqf

added more fitting SAM and radar, deleted MTLB with strelas because it caused issues

Signed-off-by: essem-451 <g1234500000p@gmail.com>

wersal454 committed:
Update A3A/addons/core/functions/Missions/fn_LOG_Airdrop.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/scrt/Encounter/fn_encounter_SpecOpsAirdrop.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/core/Templates/Templates/DLC_content/vehicles/WS/Vanilla_AAF.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/core/Templates/Templates/DLC_content/vehicles/WS/Vanilla_CSAT_Arid.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/core/Templates/Templates/DLC_content/vehicles/WS/Vanilla_NATO&AAF.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/core/Templates/Templates/DLC_content/vehicles/WS/Vanilla_NATO&LDF_Temparate.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update A3A/addons/core/Templates/Templates/DLC_content/vehicles/WS/Vanilla_NATO&LDF_Arid.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
missing textures

wersal454 committed:
small plane config fix

Signed-off-by: wersal <wroma@bk.ru>

Azodion committed:
flagTexture changes

Azodion committed:
NorAF Temperate Update

the forwarded message in dev-chat said the helmets may be hidden but i had no issues with that. the class names for some uniforms changed though. which is the opposite of what was said in the forwarded message Xd

i did change the helmets to include the new MC variants in the Elite and SOF entries.

the Arctic variant of NorAF seemed to not have been affected by the update thankfully. so I didn't have to touch that.

wersal454 committed:
4x4 logistics adjustments

Silence-112 committed:
Add: Missing groupPoliceOfficers

wersal454 committed:
more 4x4 logistics adjustments

Yes, I'm picky like that

UnseenKill committed:
Added logistics nodes to BWMod vehicles where appropriate

UnseenKill committed:
Switched to model-based config, corrected nodes

UnseenKill committed:
Added logistics nodes to MH-9M and its submodels

UnseenKill committed:
Dead variable. Never read, only written to.

UnseenKill committed:
Make this a proper function

UnseenKill committed:
Ensure "allow/prevent AIs from using vehicle" actions are set for vehicles/statics in watchposts etc.

UnseenKill committed:
Only collect unique vehicles first, then apply save properties

UnseenKill committed:
Introduce temporary resting place for hardcoded variables

UnseenKill committed:
New helper utility functions for marker queries

UnseenKill committed:
Use new marker queries, optimize inclusion conditions

UnseenKill committed:
Function renamed

UnseenKill committed:
Event listener to trigger crewing of statics/vehicles near emplacements (AA, AT, HMG, etc.)

UnseenKill committed:
Post code-review changes

UnseenKill committed:
Added scanHorizon to AT emplacements, replaced 300 hardcode with variable

UnseenKill committed:
Outpost crews can scan horizons, too

UnseenKill committed:
Turns out, `scanHorizon` scans things above the horizon; namely the sky. Fix it, so non A/A units stop staring at the skies. Also increase sleep delay, so turrets don't look so nervous in the service.

UnseenKill committed:
No such function `BIS_func_turretPath`, also returned values of `assignedVehicleRole` are supposed to be lower case and turret is never alone.

UnseenKill committed:
Vehicles w/ gunner & commander will only scan the horizon from commander seat

UnseenKill committed:
Wait until spawned location has "settled" before crewing

UnseenKill committed:
Upped priority according to PR suggestion

UnseenKill committed:
Move teardown actions assignment to its own function, add to respawn logic

UnseenKill committed:
Update A3A/addons/core/functions/Base/fn_updateRebelStatics.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: UnseenKill <post@oliver-schieche.de>

UnseenKill committed:
Update A3A/addons/core/functions/Base/fn_updateRebelStatics.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: UnseenKill <post@oliver-schieche.de>

UnseenKill committed:
Comment no longer accurate

UnseenKill committed:
Still need the vehicle radar thing to run

UnseenKill committed:
No looking around when static is inside bunker, guard post etc.

UnseenKill committed:
Player variable persists through respawn; blocks new actions to be added

jwoodruff40 committed:
fix missing bracket

UnseenKill committed:
Update A3A/addons/logistics/Nodes/RHS.hpp

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: UnseenKill <post@oliver-schieche.de>

UnseenKill committed:
Node was way too high

UnseenKill committed:
This might be important

UnseenKill committed:
Introduce "canCouple" node flag to allow for multiple cargo planes

UnseenKill committed:
Disallow node coupling on node two. Two small items is all you get.

UnseenKill committed:
Block cargo seat #0

UnseenKill committed:
Fixed nodes on civilian, armored SUV 4x4, decoupled slightly z-fixed nodes on SUV 4x4 Armed

UnseenKill committed:
Fix after test

UnseenKill committed:
Final fix on armored SUV 4x4 whose model is kinda offset

UnseenKill committed:
Logic optimization

wersal454 committed:
Update A3A/addons/core/functions/UtilityItems/fn_initObjectRemote.sqf

Co-authored-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>
Signed-off-by: wersal <wroma@bk.ru>

jwoodruff40 committed:
Update A3A/addons/core/Templates/Templates/RACS/templates.hpp

Signed-off-by: John Woodruff <22091381+jwoodruff40@users.noreply.github.com>

wersal454 committed:
Update fn_addVehicle.sqf

wersal454 committed:
Update fn_addVehicle.sqf

wersal454 committed:
Update fn_addVehicle.sqf

wersal454 committed:
Update fn_addVehicle.sqf

wersal454 committed:
Merge branch 'garage-vehicles-to-rebel-outposts' of https://github.com/wersal454/A3-Antistasi into garage-vehicles-to-rebel-outposts

wersal454 committed:
Update fn_categoryOverrides.sqf

wersal454 committed:
Update fn_itemArrayWeight.sqf

wersal454 committed:
Update fn_flagaction.sqf

wersal454 committed:
merge conflict fixies

wersal454 committed:
Update fn_addVehicle.sqf

wersal454 committed:
merge conflict fixes

wersal454 committed:
merge conflict fixies

wersal454 committed:
Persistence bitcheeees

wersal454 committed:
Merge branch 'unstable' into garage-vehicles-to-rebel-outposts

Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
UI fixes

wersal454 committed:
locality error fix
